discord/sinks/core.py

```diff
@@ -0,0 +1,210 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2015-2021 Rapptz
```

Comment:
Why is this here? Has this file been moved from somewhere else?

Reply:
wdym by "moved"?

Reply:
Is it taken/copied from somewhere else?

Reply:
it is a enhanced version of the discord.py pull as said from the original PR https://github.com/Pycord-Development/pycord/pull/532

Reply:
I see